hawkular-command-gateway-war/src/main/java/org/hawkular/cmdgw/ws/command/CommandContext.java

```diff
@@ -19,30 +19,15 @@
 import javax.jms.ConnectionFactory;
 import javax.websocket.Session;
 
-import org.hawkular.cmdgw.ws.server.ConnectedFeeds;
-import org.hawkular.cmdgw.ws.server.ConnectedUIClients;
-
 public class CommandContext {
-    private final ConnectedFeeds connectedFeeds;
-    private final ConnectedUIClients connectedUIClients;
     private final ConnectionFactory connectionFactory;
     private final Session session;
 
-    public CommandContext(ConnectedFeeds f, ConnectedUIClients ui, ConnectionFactory cf, Session session) {
-        this.connectedFeeds = f;
-        this.connectedUIClients = ui;
+    public CommandContext(ConnectionFactory cf, Session session) {
```

Comment:
the context was supposed to be useful for listeners. The context was supposed to be able to give access to other endpoints (feeds, ui clients) so listeners can send things to other endpoints. The way this is, we've removed that ability to be used in the future.